spring-cloud-config-monitor/src/main/java/org/springframework/cloud/config/monitor/FileMonitorConfiguration.java

```diff
@@ -185,17 +186,19 @@ public void poll() {
 	}
 
 	private Set<Path> getFileRepo() {
-		if (this.scmRepository != null) {
+		if (this.scmRepositories != null) {
+			String repositoryUri = null;
 			try {
-
-				Resource resource = this.resourceLoader
-						.getResource(this.scmRepository.getUri());
-				if (resource instanceof FileSystemResource) {
-					return Collections.singleton(Paths.get(resource.getURI()));
+				for (AbstractScmEnvironmentRepository repository : scmRepositories) {
+					repositoryUri = repository.getUri();
+					Resource resource = this.resourceLoader.getResource(repositoryUri);
+					if (resource instanceof FileSystemResource) {
+						return Collections.singleton(Paths.get(resource.getURI()));
```

Comment:
I guess you only watch the first one? What if there are multiple `file://` git repos?